packages/itwin/grouping-mapping-widget/src/components/SharedComponents/DeleteModal.tsx

```diff
@@ -27,28 +27,27 @@ export const DeleteModal = ({
   onDelete,
   refresh,
 }: DeleteModalProps) => {
-  const [isDeleting, setIsDeleting] = useState<boolean>(false);
+  const [localEntityName] = useState(entityName);
```

Comment:
What's the purpose of this state? Can't you just use the string from props?

Reply:
It's a copy of the original string. When the deletion happens, for a brief moment it becomes and empty string.

Reply:
Why does this become an empty string?

Reply:
Because the original array holding the reference to this string gets updated and that reference disappears when the list updates and the entity is gone. The modal has a closing animation, so while it closes you can see the string disappear and become empty for a brief second.